packages/ui/src/Toggle.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC, FocusEvent, ChangeEvent, useRef } from 'react'
+import cn from 'classnames'
+import { Error, errorId } from './Error'
+import { Help, helpTooltipId } from './Help'
+import { useUID } from 'react-uid'
+import { DataTestProp } from '@hazelcast/helpers'
+import styles from './Toggle.module.scss'
+
+export type ToggleCoreProps = {
+  name: string
+  value?: string
+  checked?: boolean
+  onBlur?: (e: FocusEvent<HTMLInputElement>) => void
+  onChange?: (e: ChangeEvent<HTMLInputElement>) => void
+  error?: string
+
+  label: string | React.ReactNode
+  helperText?: string
+  disabled?: boolean
+  className?: string
+  classNameLabel?: string
+}
```

Comment:
Core props are the ones that are passed by `formik`. Extra props are everything else we need. So I would suggest to split it like this.
```suggestion
type ToggleCoreProps = {
  name: string
  value?: string
  checked?: boolean
  onBlur?: (e: FocusEvent<HTMLInputElement>) => void
  onChange?: (e: ChangeEvent<HTMLInputElement>) => void
  error?: string
}
export type ToggleExtraProps = {
  label: string | React.ReactNode
  helperText?: string
  disabled?: boolean
  className?: string
  classNameLabel?: string
}
type ToggleProps = ToggleCoreProps & ToggleExtraProps
```
Now We can use only `ToggleExtraProp` in `ToggleFormik`

Reply:
done.